Nexd/Nexd/ui/onboarding/RegistrationViewController.swift

```diff
@@ -179,15 +189,36 @@ class RegistrationViewController: ViewController<RegistrationViewController.View
             make.width.equalTo(24)
             make.right.equalToSuperview().offset(-41)
         }
+        contentView.addSubview(privacyPolicy)
+        privacyPolicy.backgroundColor = .clear
+        privacyPolicy.isScrollEnabled = false
+        privacyPolicy.textContainerInset = .zero
+
+        let term = R.string.localizable.registration_term_privacy_policy()
+        let formatted = R.string.localizable.registration_label_privacy_policy_agreement(term)
+        privacyPolicy.attributedText = formatted.asLink(range: formatted.range(of: term), target: "https://www.nexd.app/privacypage")
```

Comment:
the URL seems to be no longer valid... new URL for the privacy page is: https://www.nexd.app/privacy